spec/controllers/admin/categories_controller_spec.rb

```diff
@@ -16,6 +16,13 @@
     assert_response :redirect, :action => 'index'
   end
 
+  it "test_new" do
```

Comment:
Flash notice not displaying bug has been fixed. Cucumber test written and passing to check for failure case.